stock_manual_transfer/security/res_groups.xml

```diff
@@ -3,6 +3,7 @@
 
     <record id="group_stock_manual_transfer" model="res.groups">
         <field name="name">Can make manual transfers</field>
+        <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]" />
```

Comment:
Use the Command syntax, please

Reply:
@luisg123v I make the change, is it correct how I did it?

Reply:
Not applied correctly, I meant:

`<field name="users" eval="[Command.link(ref('base.user_root')), Command.link(ref('base.user_admin'))]" />`

For more info, read [this](https://github.com/orgs/OCA/discussions/157#discussioncomment-8753910)